inst/extdata/OSD/S/SQUALICUM.json

```diff
@@ -124,6 +160,24 @@
         "distinctness": "NA",
         "topography": "NA",
         "narrative": "2Cr--44 to 60 inches; olive gray (5Y 5/2) dense glacial till that breaks to gravelly loam, light gray (5Y 7/2) dry; massive; very hard, firm, nonsticky, nonplastic; 30 percent hard pebbles and 5 percent weathered pebbles; NaF pH 10.0; moderately acid (pH 5.7)."
+      },
+      {
+        "name": "Oi",
+        "top": "NA",
+        "bottom": "NA",
+        "dry_hue": "NA",
+        "dry_value": "NA",
+        "dry_chroma": "NA",
+        "moist_hue": "NA",
+        "moist_value": "NA",
+        "moist_chroma": "NA",
+        "texture_class": "NA",
+        "cf_class": "NA",
+        "pH": "NA",
+        "pH_class": "NA",
+        "distinctness": "NA",
+        "topography": "NA",
+        "narrative": "Oi--l inch to 0; undecomposed needles, leaves and twigs."
```

Comment:
Not sure why at least the bottom depth isnt parsed here, but at least a missing horizon has been identified